### DISCUSSION TITLE
Width issues with sortable tables

### DISCUSSION THREAD

**Discussion by captenmasin** [livewire/livewire#1184] [4. General]
Hi all

Has anyone else come across the problem of having a table with `wire:sortable` looking all sorts of messed up width-wise when sorting?

[Giphy example](http://www.giphy.com/gifs/MEjUWBQWpFdczJdPLP)

If so, how did you overcome it?

**Comment by mblarsen** [comment]
The Giphy example is set to private.

So I can only assume that you mean when dragging the dragged part shrinks or otherwise changes light out?

That happens because it is not a child of the original elements parent any more but explicitly positioned. So it will no longer get an automatic with from being flexed or in a grid or any other layout.

One way to fix it is to set a fixed with to your element.

Another is to purposely change the look and feel of what you are dragging to let the user know that is sort of a placeholder and not the real thing. 

I implemented a Trello style board for a client. I set the card width fixed, but in additon I added padding, shadows and rounded boards to the card being dragged.

**Reply by MasonSpeed** [reply]
Whoops my bad!

[New Giphy link](https://media.giphy.com/media/f4aBG9JjCsv6xqmPzi/giphy.gif)

But yes you are absolutely right... I like the idea of styling it completely different I think I'll give that a go

Thanks!
